scylla-gdb.py

```diff
@@ -1797,9 +1797,7 @@ def _lookup_type(type_names):
 
     types = []
     try:
-        # For Scylla < 2.1
-        # FIXME: this only finds range readers
-        types = [_lookup_type(['sstable_range_wrapping_reader'])]
+        return intrusive_list(gdb.parse_and_eval('sstables::_reader_tracker._readers'), link='_tracker_link')
```

Comment:
Don't we need to fall back to `types = [_lookup_type(['sstable_range_wrapping_reader'])]` in case the intrusive_list access raises gdb.error?

The reason for backward compatibility requirement is that we want the latest and greatest scylla-gdb to work also with released binaries, not only in master.

Reply:
@denesb please review too

Reply:
The old path was for "Scylla < 2.1", which is so old that I figured we can drop it.